1-BasicSetUp.sh

```diff
@@ -57,7 +57,11 @@ cp ./config_files/.tmux.conf ~
 cp ./config_files/.tmux.conf.local ~
 mkdir -p ~/.config/tilda
 cp ./config_files/config_0 ~/.config/tilda/
-
+#Checks if ZSH is partially or completely Installed to Remove the folders and reinstall it
+zsh_folder=/opt/.zsh/
+if [[ -d $zsh_folder ]];then
+	rm -r /opt/.zsh/*
```

Comment:
This needs superuser access, so prefix this with `sudo`

Reply:
I have prefixed the remove command with `sudo`